internal/providers/cmdline/cmdline.go

```diff
@@ -31,10 +31,6 @@ import (
 	"github.com/coreos/ignition/internal/resource"
 )
 
-const (
```

Comment:
I'd prefer to keep these defined up here. Maybe define something like `cmdlineUrlFlagLegacy` and `cmdlineUrlFlag`?

Reply:
Will fix and update